bin/simple_engine.ml

```diff
@@ -16,31 +16,32 @@ let runtime_counter device tones _domain_id ts counter _value =
   match counter with
   | EV_C_MINOR_PROMOTED ->
       starting_time := Some ts;
-      Midi.(write_output device [ message_on ~note:(tones 0) ~timestamp:0l () ])
+      Midi.(
+        write_output device
+          [
+            message_on ~note:(tones 0).Scale.note ~timestamp:0l
```

Comment:
You could do the same here and bind `note` and `volume` using a let binding statement.